var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -161,6 +163,14 @@ def configure_args(self):
                         ),
                     ]
                 )
+            else:
+                options.extend(
+                    [
+                        "WANNIER90_CPPFLAGS=-I{0}".format(spec["wannier90"].prefix.modules),
+                        "WANNIER90_LIBS=-L{0}".format(spec["wannier90"].prefix.lib),
```

Comment:
We are slowly replacing `format()` with f-strings (https://docs.python.org/3/tutorial/inputoutput.html) so suggested using that approach here and lines 188 and 190.